### DISCUSSION TITLE
Firefox width overflow problem

### DISCUSSION THREAD

**Discussion by 3282778869** [sourcelair/xterm.js#4996] [Q&A]
It will create a div with a width of 50000px under the body and in Firefox it will overflow the width and result in a horizontal scrollbar

![image](https://github.com/xtermjs/xterm.js/assets/113400444/9c0b9886-c876-465d-acb7-23ef7a3cb2f7)
![image](https://github.com/xtermjs/xterm.js/assets/113400444/6ac47cbb-885e-4cfa-a070-ffdeb1ef087c)

**Comment by slivingston** [comment]
fixed in https://github.com/xtermjs/xterm.js/pull/4807
